examples/openid-connect/link.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
You can run this script to easily run the example. Once we release v6.4, this will be no longer needed.